doc/radosgw/notifications.rst

```diff
@@ -98,6 +98,8 @@ Request parameters:
   - "none": message is considered "delivered" if sent to broker
   - "broker": message is considered "delivered" if acked by broker (default)
 
+.. tip:: The topic-name (see :ref:`radosgw-create-a-topic`) is used for the AMQP "routing key"
```

Comment:
just want to be clear that we assume that the exchange is a "topic exchange", maybe better to say:
```
The topic-name (see :ref:`radosgw-create-a-topic`) is used for the AMQP topic
```
or:
```
The topic-name (see :ref:`radosgw-create-a-topic`) is used for the AMQP topic (the "routing key" for a topic exchange)
```
see [here](https://www.rabbitmq.com/tutorials/tutorial-five-python.html)